crates/bevy_gltf/src/loader.rs

```diff
@@ -479,14 +493,23 @@ async fn load_gltf<'a, 'b, 'c>(
                 && primitive.material().normal_texture().is_some()
             {
                 bevy_log::debug!(
-                    "Missing vertex tangents, computing them using the mikktspace algorithm"
+                    "Missing vertex tangents for {}, computing them using the mikktspace algorithm. Use Blender to bake this information into the gltf", file_name
                 );
+
+                let start = Instant::now();
```

Comment:
Thinking about it, in my opinion it is better to remove the use of `Instant` and the `warn!` below, and rather wrap the block below which computes tangents in a `span!` which mentions the file name.

This way profilers can pick up how long it takes and aggregate it etc.

Also a warning would be a bit too heavy-handed anyway since some users might be ok with having them auto-computed.

Reply:
Did you have something like this in mind?

Reply:
Yes- but perhaps using https://docs.rs/tracing/latest/tracing/span/struct.Span.html#method.in_scope
to avoid the explicit drop/exit

Reply:
Gotcha!